tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example_raw_scalars/static/views.js

```diff
@@ -13,17 +13,16 @@
 // limitations under the License.
 // =============================================================================
 
-import * as Model from './model.js';
+// Generic view builders.
 
 /**
- * @param {string} run
+ * @param {!Map<string, !Array<Object>>} tagsToScalars
  * @return {!Promise<!DocumentFragment>}
  */
-export async function createPreviews(run) {
+export async function createPreviews(tagsToScalars) {
```

Comment:
Remove async. No more await and it does not make sense for view to be created asynchronously anymore.